### PR TITLE
CEDS-2189 Make CommodityDetails.descriptionOfGoods optional

### PR DESCRIPTION
**Pull request by PauloPortugal**
For Clearance Requests, data elements '6/14 Commodity code' and
'6/8 Description of Goods' are not required in the majority of cases.

This data element is not required on clearance requests using procedure
codes 00 18 or 00 19 in D.E. 1/10.

**Depends on: https://github.com/hmrc/customs-declare-exports/pull/169**

### CODE REVIEW

File: app/forms/declaration/CommodityDetails.scala [hunk 1]
```diff
@@ -23,7 +23,7 @@ import play.api.data.{Form, Mapping}
 import play.api.libs.json.Json
 import utils.validators.forms.FieldValidator._
 
-case class CommodityDetails(combinedNomenclatureCode: Option[String], descriptionOfGoods: String)
+case class CommodityDetails(combinedNomenclatureCode: Option[String], descriptionOfGoods: Option[String])
```
Comment (SteveSugden) [inline app/forms/declaration/CommodityDetails.scala]: I think I prefer this way of creating optional & required mappings.  The ones I have done so far use different apply/unapply methods (idea borrowed from `TotalPackageQuantity`)

File: app/views/declaration/summary/item_section.scala.html [hunk 2]
```diff
@@ -14,7 +14,10 @@
  * limitations under the License.
  *@
 
-@import models.DeclarationType._
+@import views.html.declaration.summary.onward_supply_relief
+@import views.html.declaration.summary.package_information
+@import views.html.declaration.summary.union_and_national_codes
```
Comment (SteveSugden) [inline app/views/declaration/summary/item_section.scala.html]: Why all the new imports?

Comment (PauloPortugal) [reply app/views/declaration/summary/item_section.scala.html]: Twirl templates without explicit view imports within IntelliJ Ultimate are flagged as errors.
If we explicitly import them this goes away.
I know that this is meaningless

Comment (SteveSugden) [reply app/views/declaration/summary/item_section.scala.html]: That is a useful tip!

**Comment by PauloPortugal** [comment]
Smoke tests have passed as well

**Pull request merged**